test/src/api/responderTest.php

```diff
@@ -4559,10 +4559,6 @@ public function test_interface_game_009() {
             $gameId, 'add', 5);
         $_SESSION = $this->mock_test_user_login('responder003');
 
-        // expected changes
-        // #1266 - the API should lie about this to avoid information leaks
-        $expData['playerDataArray'][1]['waitingOnAction'] = FALSE;
```

Comment:
While you're changing here, maybe replace the comment with one noting that there should be no expected changes, so later regressions will jump out at us more?